src/components/TypeFaceSubFamilies/_typeface-sub-families.scss

```diff
@@ -70,6 +70,21 @@
   padding-bottom: 2.5rem;
   margin-bottom: 0;
   display: flex;
+  flex-direction: column;
+  font-size: calc(2rem + 0.25 * (100vw - 20rem) / 22);
+
+  @include carbon--breakpoint('md') {
+    flex-direction: row;
+    font-size: calc(2.25rem + 0.375 * (100vw - 42rem) / 24);
+  }
+
+  @include carbon--breakpoint('lg') {
+    font-size: calc(2.625rem + 0.375 * (100vw - 66rem) / 16);
+  }
+
+  @include carbon--breakpoint('xlg') {
+    font-size: calc(3rem + 0.75 * (100vw - 82rem) / 17);
```

Comment:
I think this should be 
`@include carbon--type-style('expressive-heading-05', true);`

??

Reply:
I'm not familiar with that mixin but it looks like it includes all of the media query rules already? (see latest commit)